src/classes/AwsConstruct.ts

```diff
@@ -29,5 +30,5 @@ export abstract class AwsConstruct extends CdkConstruct implements ConstructInte
     /**
      * CloudFormation references
      */
-    abstract references(): Record<string, Record<string, unknown>>;
+    abstract references(): Record<string, AwsCfInstruction>;
```

Comment:
These changes have been merged on master, it looks like a rebase problem?

Reply:
Wasn't aware of that, I applied those changes my self. I will try to rebase.

Reply:
I don't see those on master. It should be here for example https://github.com/getlift/lift/blob/master/src/classes/AwsConstruct.ts#L32

Let me know if I'm missing something.

Reply:
🤦  my bad those were in the `db` branch. All good!